src/metabase/driver/common.clj

```diff
@@ -217,6 +217,24 @@
    :description  (deferred-tru "This enables Metabase to scan for additional field values during syncs allowing smarter behavior, like improved auto-binning on your bar charts.")
    :visible-if   {"advanced-options" true}})
 
+(def auth-basic-token
+  "Map representing the authentication token option in a Druid DB connection."
```

Comment:
Seeing how these are Druid specific, I don't think it's appropriate to put them in `driver.common`. I'd just put these directly in the YAML and not in the Clojure code at all